src/app.py

```diff
@@ -475,4 +467,4 @@ def update_output(cat, rate, year):
 
 
 if __name__ == '__main__':
-    app.run_server(debug = True)
\ No newline at end of file
+    app.run_server(debug = True)
```

Comment:
I would suggest we remove the debug option

Reply:
Let us remove it in the final merge to main branch.

Reply:
Can you merge this PR? @thayeylolu